qa/suites/orch/cephadm/mgr-nfs-upgrade/2-nfs.yaml

```diff
@@ -0,0 +1,30 @@
+tasks:
+
+# stop kernel nfs server, if running
+- vip.exec:
+    all-hosts:
+      - systemctl stop nfs-server
+
+- cephadm.shell:
+    host.a:
+      - ceph fs volume create foofs
+
+- cephadm.wait_for_service:
+    service: mds.foofs
+
+- cephadm.shell:
+    host.a:
+      - ceph nfs cluster create cephfs foo --placement=2
+      - ceph nfs export create cephfs foofs foo --binding /fake
```

Comment:
`binding` -> `pseudo_path`

Reply:
this test is installing an old version (with --binding) and then upgrading